vendor/engines/bulk_email/app/views/bulk_email/bulk_email/_fields.html.haml

```diff
@@ -2,17 +2,27 @@
 
   .row
     .span3
-      = f.input :search_type, collection: @search_types.invert, selected: params[:search_type], input_html: { name: "search_type" }
+      - @user_types.each do |user_type, label_text|
+        = label_tag(user_type) do
+          = check_box_tag "bulk_email[user_types][]",
```

Comment:
I did not find a way to get `collection_check_boxes` to generate what this does.